examples/stocks/lib/stock_symbol_viewer.dart

```diff
@@ -82,7 +77,17 @@ class StockSymbolPage extends StatelessWidget {
         children: <Widget>[
           new Container(
             margin: new EdgeInsets.all(20.0),
-            child: new Card(child: new StockSymbolView(stock: stock))
+            child: new Card(
+              child: new _StockSymbolView(
+                stock: stock,
+                arrow: new Hero(
+                  key: new ObjectKey(stock),
+                  tag: stock,
+                  turns: 2,
+                  child: new StockArrow(percentChange: stock.percentChange)
```

Comment:
since you're passing in the stock anyway, why not do this in the StockSymbolView builder?